<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flashcardinput</class>
 <widget class="QMainWindow" name="flashcardinput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>flashcardinput</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="inputWord">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 218, 218);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputMeaning">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>690</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bgflashc/btnsave.png);

font: 900 10pt &quot;Swis721 Blk BT&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>690</y>
      <width>179</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 900 10pt &quot;Swis721 Blk BT&quot;;

background-image: url(:/bgflashc/btncon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bgflashc/bgflashin.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="saveStatus">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>710</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 0, 0);
font: 12pt &quot;MS PGothic&quot;;</string>
    </property>
    <property name="text">
     <string>saveStatus</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>inputWord</zorder>
   <zorder>inputMeaning</zorder>
   <zorder>saveButton</zorder>
   <zorder>continueButton</zorder>
   <zorder>saveStatus</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
